app/src/main/java/com/kickstarter/viewmodels/projectpage/CheckoutFlowViewModel.kt

```diff
@@ -31,18 +29,16 @@ data class FlowUIState(
 class CheckoutFlowViewModel(val environment: Environment) : ViewModel() {
```

Comment:
Do we have tests associated with this ViewModel?

Reply:
not yet

Reply:
Not for AddOnsVM either. Will add them as a followup ticket not in this PR

Reply:
Sorry folks, but I do not feel comfortable not having VM's tested, gonna have to do a request changes here

Reply:
We will have tickets for fast follow to add tests to the VM's